conda-env/dev-nompi.yml

```
@@ -26,7 +26,9 @@ dependencies:
   - matplotlib-base
   - netcdf4
   - output_viewer >=1.3.0
-  - numpy >=1.23.0
+  - numpy >=2.0.0,<3.0.0
+  - shapely >=2.0.0,<3.0.0
+  - xarray >=2024.03.0
   - shapely >=2.0.0,<3.0.0
   - xarray >=2023.02.0
```

```suggestion
  - numpy >=2.0.0,<3.0.0
  - shapely >=2.0.0,<3.0.0
  - xarray >=2023.02.0
```

My fault, gave you incorrect suggestion. Merging these fixes.